src/components/Sidebar.vue

```diff
@@ -35,7 +35,11 @@
         </router-link>
       </v-toolbar-title>
       <v-spacer />
-      <login-menu />
+      <login-menu
+        :store="store"
```

Comment:
Passing the entire store seems excessive and is store implementation specific. IMO you should just pass what's needed here, the callbacks for logout and fetching user info.